src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    height: 134px;
+  }
+
+  &__title {
+    padding-top: 40px;
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: 500;
+    color: $color-main-accent;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    width: 100%;
+    padding-top: 6px;
```

Comment:
use margin, also it's better to use always `margin-bottom` for elements inside a block. it will be easier to identify how to position the next element.

Reply:
ok